pages/download.html

```diff
@@ -309,6 +309,20 @@ <h3>{% trans "Linux / Source Code" %}</h3>
   </div>
 </div>
 
+<div id="snapshots" class="docs-section container">
+  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots{% endblocktrans %}</a></h2>
```

Comment:
```suggestion
  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots (main branch){% endblocktrans %}</a></h2>
```

Reply:
I think this is unnecessary. Developers know this, and testers looking for snapshots possibly don't know what this means.

Reply:
🤷